clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/security/KubernetesNamedAccountCredentialsInitializer.groovy

```diff
@@ -99,6 +101,7 @@ class KubernetesNamedAccountCredentialsInitializer implements CredentialsInitial
           .debug(managedAccount.debug)
           .build()
 
+        kubernetesAccountConfigurer.configure(managedAccount.configuration, kubernetesAccount)
```

Comment:
I guess I'm confused why this was moved out of the "build" into a separate "configure"